layouts/partials/snippet.html

```diff
@@ -7,9 +7,15 @@
     </button>
     {{ end }}
   </div>
-  <h2>
-    {{.title}}
-  </h2>
+  {{ $page_url := .tutorial | .share_url }}
```

Comment:
hey @victoriabernard92, this line causes my local version of the branch to not build

```
~/src/workers/sites/workers-docs victoria/template_link
❯ hugo
Building sites … WARN 2019/11/27 13:34:07 Page's .Hugo is deprecated and will be removed in a future release. Use the global hugo function.
WARN 2019/11/27 13:34:07 Page's .URL is deprecated and will be removed in a future release. Use .Permalink or .RelPermalink. If what you want is the front matter URL value, use .Params.url.
WARN 2019/11/27 13:34:07 Page's .UniqueID is deprecated and will be removed in a future release. Use .File.UniqueID.
WARN 2019/11/27 13:34:07 .File.UniqueID on zero object. Wrap it in if or with: {{ with .File }}{{ .UniqueID }}{{ end }}
Total in 196 ms
Error: Error building site: "/Users/kristian/src/workers/sites/workers-docs/content/templates/_index.md:29:1": failed to render shortcode "gallery": failed to process shortcode: "/Users/kristian/src/workers/sites/workers-docs/layouts/shortcodes/gallery.html:10:30": execute of template failed: template: shortcodes/gallery.html:11:6: executing "shortcodes/gallery.html" at <partial "snippet" .>: error calling partial: "/Users/kristian/src/workers/sites/workers-docs/layouts/partials/snippet.html:10:30": execute of template failed: template: partials/snippet.html:10:30: executing "partials/snippet.html" at <.share_url>: share_url is not a method but has arguments
```

Reply:
Fixed 😄